<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QDialog" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fcfcfc;</string>
  </property>
  <widget class="QLineEdit" name="lineEditCurrPasswd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: #1a1a1a;
border: 1px solid;

background-color: transparent;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNewPasswd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: #1a1a1a;
border: 1px solid;

background-color: transparent;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRepNewPasswd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: #1a1a1a;
border: 1px solid;

background-color: transparent;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrPasswd">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Текущий пароль:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPasswd">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Новый пароль:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRepNewPasswd">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>180</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Повторите новый пароль:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChangePasswd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1a1a1a;

border: none;
color: #fcfcfc;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Сменить пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;
color:  #1a1a1a;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMinimize">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>-4</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;
color:  #1a1a1a;</string>
   </property>
   <property name="text">
    <string>_</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
